auto_round/data_type/int.py

```diff
@@ -72,13 +72,13 @@ def double_quant_tensor(tensor, bits, q_scale_thresh):
     qdq_tensor = torch.clamp(round_ste(tensor / scale), max=maxq) * scale
     return qdq_tensor, scale
 
-def double_quant_tensor_sym(tensor, bits):
+def double_quant_tensor_sym(tensor, bits, q_scale_thresh):
     maxq = 2 ** (bits - 1)
     imax = abs(tensor).argmax(axis=-1, keepdims=True)
     wmax = torch.take_along_dim(tensor, imax, axis=-1)
-    iscale = torch.where(wmax !=0, -maxq / wmax, 0)
-    scale = torch.where(iscale != 0, 1 / iscale, 0)
-    qdq_tensor = torch.clamp(round_ste(tensor * iscale), min=-maxq, max=maxq-1) * scale
+    scale = wmax / -maxq
+    scale = torch.where(scale < 0, torch.clamp(scale, max=-q_scale_thresh), torch.clamp(scale, min=q_scale_thresh))
```

Comment:
if the scale is always positive, no need to handle like this. Besides, it's mainly for fp16 datatype. If it's fp32 or bf16, I think handing zero scale is enough